.github/workflows/merge-when-ready.yml

```diff
@@ -96,7 +96,7 @@ jobs:
                   chmod uog+x target/release/container-chain-simple-node
                   chmod uog+x target/release/container-chain-frontier-node
 
-            - name: Run Zombienet Test ${{ matrix.test_name }}
```

Comment:
why removing the matrix name?

Reply:
because it's not a matrix

Reply:
wait how is it not? https://github.com/moondance-labs/tanssi/blob/3d9025a271799ff917455670ff4657a3ac9179bb/.github/workflows/merge-when-ready.yml#L47